tests/Unittest-README.md

```diff
@@ -0,0 +1,159 @@
+# Ansible CVP Unit testing
+
+## Unit test structure
+The structure of unittests are described below
+- **`tests/unit`**: Contains all unittests 
+  - Filename start with `test_` followed by the name of the file it's testing. 
+    - For eg: Unittests for `ansible_collection/arista/cvp/plugins/module_utils/device_tools.py` will be under `tests/unit/test_cv_device_tools.py`
+  - `tests/unit/conftest.py`: This is a special python script that defines all the fixtures available for use while testing `ansible-cvp` modules
+- **`tests/lib`**: Contains library functions
+  - **`tests/lib/mockMagic.py`**: Contains mock cvprac functions
+    - Note: the name and signature of the mock functions matches original cvprac functions
+- **`tests/data`**: Contains faked datasets for use by the mocked cvprac functions
+
+## Write unittests
+### General guidelines
+- Create your unittest file under `tests/unit` following the naming convention mentioned in the [above section](#unit-test-structure)
```

Comment:
In lists, try not to mix and match sentence and non sentence structure. Stick to one or the other in the same list